.github/workflows/railway-status-check.yml

```
@@ -0,0 +1,113 @@
+          else
+            echo "⚠️  Application returned: $RESPONSE"
+            echo "status=unhealthy" >> $GITHUB_OUTPUT
+          fi
+
+      # ======================================================================
+      # 3. Check database connectivity
+      # ======================================================================
+      - name: 🗄️  Check database connectivity
+        id: db-check
+        continue-on-error: true
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:8:30: Double quote to prevent globbing and word splitting

(shellcheck)